extensions/sql-bindings/src/services/azureFunctionsService.ts

```diff
@@ -26,6 +26,8 @@ export async function createAzureFunction(node?: ITreeNodeInfo): Promise<void> {
 	let connectionInfo: IConnectionInfo | undefined;
 	let isCreateNewProject: boolean = false;
 	let newFunctionFileObject: azureFunctionsUtils.IFileFunctionObject | undefined;
+	let connectionURI: string = '';
```

Comment:
You only need this for verifying the connection so just declare it down right before the withProgress call